build.assets/tooling/cmd/notarize-apple-binaries/gonWrapper.go

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
This probably wants to be renamed `gon_wrapper.go` in ths same fashion as `gon_config.go` was

Reply:
Updated